sonic_platform_base/sonic_sfp/qsfp_dd.py

```diff
@@ -46,7 +46,10 @@ def decode_module_state(self, eeprom_data, offset, size):
 
     def decode_connector(self, eeprom_data, offset, size):
         connector_id = eeprom_data[offset]
-        return connector_dict[connector_id]
+        if connector_id in connector_dict.keys():
```

Comment:
can we put a similar fix for https://github.com/Azure/sonic-platform-common/blob/4533f829781f9d3706406f33d82a58adef74d294/sonic_platform_base/sonic_sfp/qsfp_dd.py#L77?. So,that the application crash can be avoided?.

Reply:
> can we put a similar fix for
> 
> https://github.com/Azure/sonic-platform-common/blob/4533f829781f9d3706406f33d82a58adef74d294/sonic_platform_base/sonic_sfp/qsfp_dd.py#L77
> 
> ?. So,that the application crash can be avoided?.

added fix